backend/index.js

```diff
@@ -2,6 +2,7 @@ import mongoose from 'mongoose';
 import express from 'express';
 import bodyParser from 'body-parser';
 import dotenv from 'dotenv';
+import cors from 'cors';
```

Comment:
Had to allow CORS because we test backend and frontend on different origins. Can be changed during deployment